src/components/booklist.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Book from './book';
+
+const BOOKDATA = [
+  {
+    id: 1,
+    title: 'The Hunger Game',
+    author: 'Suzanne Collins',
+    category: 'action',
+  },
+  {
+    id: 2,
+    title: 'Dune',
+    author: 'Frank Herbert',
+    category: 'science fiction',
+  },
+  {
+    id: 3,
+    title: 'Capital in the Twenty-First Century',
+    author: 'Suzanne Collins',
+    category: 'economy',
+  },
+];
+
+const Booklist = () => (
+  <div>
+    <ul>
+      {BOOKDATA.map((book) => (
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        <Book {...book} />
+      ))}
```

Comment:
- [ ] Kindly solve this linter error instead of disabling it.
- [ ] Also in react when using the map method on an array you should add the `key` property that uniquely identifies each rendered element. so that the virtual dom gets references to the added elements. Because of that currently, you have an error in your console. :point_down: 

![image](https://user-images.githubusercontent.com/65068771/156447291-29d0387c-e551-4299-b352-5bffddd5fd65.png)